tests/codex/blockexchange/engine/testadvertiser.nim

```diff
@@ -22,6 +22,7 @@ asyncchecksuite "Advertiser":
     blockDiscovery: MockDiscovery
     localStore: BlockStore
     advertiser: Advertiser
+    advertised: seq[Cid]
```

Comment:
That's right. It happens only rarely. I tried running the tests with 0 workers, that also fixed it. But I figured it would be better to include the workers in the test.